TeamCode/src/main/java/org/firstinspires/ftc/teamcode/commands/groups/SequentialShooter.java

```diff
@@ -8,21 +8,18 @@
 
 public class SequentialShooter extends SequentialCommandGroup {
 
-    private InstantCommand runFlyWheelCommand;
-    private WaitCommand waitCommand;
     private Com_Shooter shooterCommand;
 
-    public SequentialShooter(InstantCommand commandOne, WaitCommand commandTwo,
-                             Com_Shooter commandThree) {
-        runFlyWheelCommand = commandOne;
-        waitCommand = commandTwo;
-        shooterCommand = commandThree;
+    public SequentialShooter(InstantCommand runFlyWheelCommand, WaitCommand waitCommand,
+                             Com_Shooter shooterCommand) {
+        this.shooterCommand = shooterCommand;
 
         addCommands(runFlyWheelCommand, waitCommand, shooterCommand);
     }
 
     @Override
     public void end(boolean interrupted){
         shooterCommand.stopShooter();
+        shooterCommand.resetFlicker();
```

Comment:
that doesn't return it to the resting position, it just resets the timer and sets it to `RUNNING` mode.

Reply:
You need to actually set the position of the servo to the resting the position directly.